src/models/user.ts

```diff
@@ -72,10 +73,16 @@ const UserSchema = new mongoose.Schema(
   },
 );
 
-UserSchema.virtual('event', {
-  ref: 'Event',
+UserSchema.virtual('position', {
+  ref: ModelRefs.POSITION,
   localField: '_id',
-  foreignField: 'owner',
+  foreignField: 'occupant',
```

Comment:
Should this be `justOne: true`?

Reply:
I think the schema of `Position` suggests that there will only ever be one occupant. 
What does `justOne: true` do?

Reply:
It'll make the virtual `position` property on User docs a single doc or null. As documented [here](https://mongoosejs.com/docs/api/schema.html#schema_Schema-virtual).

Reply:
Yeaa this makes sense. updated in 775aec3f4620d07c84cc9b51685b62bcfd93f732